### ISSUE TITLE
beta12 import error

### ISSUE THREAD

**Issue by wieseljonas** [urish/angular2-moment#25]
hi I'm struggling after moving to angular2 beta12 

```
 ✗ TypeScript error: /xxxx/node_modules/angular2-moment/CalendarPipe.ts(49,7): Error TS2322: Type 'EventEmitter<{}>' is not assignable to type 'EventEmitter<Date>'.
  Type '{}' is not assignable to type 'Date'.
    Property 'toDateString' is missing in type '{}'.
 ✗
/xxxxx/node_modules/angular2-moment/TimeAgoPipe.ts:3
import {Pipe, ChangeDetectorRef, PipeTransform, OnDestroy} from 'angular2/core';
^
```

**Comment by urish** [comment]
Hi, can you please publish a small code sample that reproduces the issue? Gist or plunker.

**Comment by wieseljonas** [comment]
Hi Urish here is my package.json . I get the error when I declare the TimeAGoPipe in my Component.

```
{
  "name": "mako",
  "description": "Mako",
  "license": "Apache-2.0",
  "repository": {
    "type": "git",
    "url": "https://github.com/driftyco/ionic-conference-app.git"
  },
  "dependencies": {
    "angular2": "^2.0.0-beta.7",
    "angular2-jwt": "0.1.6",
    "es6-promise": "3.0.2",
    "es6-shim": "^0.35.0",
    "ionic-angular": "2.0.0-beta.3",
    "ionic-native": "^1.1.0",
    "ionicons": "3.0.0-alpha.3",
    "lodash": "^4.6.1",
    "reflect-metadata": "0.1.2",
    "rxjs": "^5.0.0-beta.2",
    "zone.js": "^0.6.6"
  },
  "devDependencies": {
    "babel-preset-es2015": "^6.6.0",
    "del": "1.1.1",
    "gulp": "3.9.1",
    "gulp-watch": "4.3.5",
    "ionic-gulp-browserify-typescript": "^1.0.0",
    "ionic-gulp-fonts-copy": "^1.0.0",
    "ionic-gulp-html-copy": "^1.0.0",
    "ionic-gulp-sass-build": "^1.0.0",
    "ionic-gulp-scripts-copy": "^1.0.0"
  },
  "cordovaPlugins": [
    "cordova-plugin-whitelist",
    {
      "locator": "https://github.com/VersoSolutions/CordovaClipboard",
      "id": "com.verso.cordova.clipboard"
    }
  ],
  "cordovaPlatforms": [
    "ios",
    "android"
  ]
}
```

**Comment by lordgreg** [comment]
I have exactly the same problem with latest ionic2 project:

```
 ✗
C:\Development\abcd\node_modules\angular2-moment\CalendarPipe.ts:3
import {Pipe, ChangeDetectorRef, PipeTransform, EventEmitter, OnDestroy} from 'angular2/core';
^
ParseError: 'import' and 'export' may appear only with 'sourceType: module'
```

**Comment by wli** [comment]
I'm seeing the same thing as @lordgreg with ionic@2.0.0-beta.24

**Comment by remithomas** [comment]
I resolved it by adding some options to the gulpfile.js specialy to buildBrowserify options. It works !

```
buildBrowserify({
 watch: true,

 browserifyOptions: { // add this
  cache: {},
  packageCache: {},
  debug: true,
  extensions: ['.js', '.ts']
 }
}).on('end', done);
```

**Comment by aramkoukia** [comment]
I get the same error when I added ng2-uploader component.
Adding the js and ts extension like above didnt fix my issue.

**Comment by urish** [comment]
should probably be fixed by #34. Please reopen if still relevant
